galogen.cpp

```diff
@@ -45,7 +45,7 @@ struct TypeInfo {
   std::string type_cdecl;
   
   // Name of another type that this type requires.
-  std::string requires;
```

Comment:
could you use `prerequisite_type` instead? now that i look at it, `requires` wasn't that good of a choice in the first place.

Reply:
sure thing.